### DISCUSSION TITLE
Hi, after setting evaluation = dict(interval=1, metric='acc') in mmocr, I found that after training 1 epoch, I can't start eval, where is the function to call up eval?

### DISCUSSION THREAD

**Discussion by htaoruan** [open-mmlab/mmocr#1585] [General]
Hi, after setting evaluation = dict(interval=1, metric='acc') in mmocr, I found that after training 1 epoch, I can't start eval, where is the function to call up eval?

**Comment by Harold-lkk** [comment]
Can you provide the full config? I can not reimplement your problem

**Reply by htaoruan** [reply]
In mmocr0.6，In which function is this configuration used evaluation = dict(interval=1, metric='acc') 。

**Reply by gaotongxiao** [reply]
We cannot reproduce this issue. Have you changed any code before? You can post an issue [following the template](https://github.com/open-mmlab/mmocr/issues/new?assignees=&labels=bug&template=1-bug-report.yml&title=%5BBug%5D+), which can help us locate the problem.
